yt_dlp/extractor/rutv.py

```diff
@@ -192,13 +191,15 @@ def _real_extract(self, url):
                         'url': url
                     }
                 fmt.update({
-                    'width': width,
-                    'height': height,
+                    'width': width * int_or_none(quality, default=height) / height,
```

Comment:
Shouldn't this be int division?
Actually if i rember correctly, in_or_none has a scale parameter...maybe that can be used here
```suggestion
                    'width': width * int_or_none(quality, default=height) // height,
```

Reply:
Fixed to use *_scale on int_or_none to calculate, since it looks like nothing hurts